apifuture/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.spotify</groupId>
+    <artifactId>futures-extra</artifactId>
+    <version>4.2.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>futures-extra-apifuture</artifactId>
```

Comment:
Perhaps name this one "futures-extra" (and name the parent something else) as to ensure that people already using "futures-extra" do not have to change their artifact name, or that you get both the old futures-extra and this one included in your classpath? You would have the last problem for -core either way though.